src/Microsoft.AspNet.SignalR.SqlServer/TaskAsyncHelper.cs

```diff
@@ -0,0 +1,1194 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
The goal is to rewrite the few places that TaskAsyncHelper is used to use async/awit, `Task.FromResult`, etc... Then you can remove this class.

I don't care if you do this in a separate PR.